src/xenia/kernel/kernel_state.cc

```diff
@@ -316,6 +316,20 @@ void KernelState::SetExecutableModule(object_ref<UserModule> module) {
     *variable_ptr = executable_module_->hmodule_ptr();
   }
 
+  // Setup the kernel's ExLoadedImageName field
+  export_entry = processor()->export_resolver()->GetExportByOrdinal(
+      "xboxkrnl.exe", ordinals::ExLoadedImageName);
+
+  if (export_entry && module) {
+    auto variable_ptr = memory()->TranslateVirtual<xe::be<uint32_t>*>(
+        export_entry->variable_ptr);
+    std::string image_name(module->path());
+    uint32_t image_line_length =
+        xe::align(static_cast<uint32_t>(image_name.length()) + 1, 256u);
```

Comment:
Why is this aligned? MAX_PATH (260), if applicable here even, is the absolute limit, not an increment.